pkg/kotsadmconfig/validation/validator.go

```diff
@@ -49,6 +49,17 @@ func isValidatableConfigItem(item kotsv1beta1.ConfigItem) bool {
 		return false
 	}
 
+	// if item is not required and value is empty, no need to validate
+	if !item.Required && item.Value.String() == "" {
+		return false
+	}
```

Comment:
moved from `pkg/kotsadmconfig/validation/config.go` so all the logic for `isValidatableConfigItem` is at one place

Reply:
can't this just be `if item.Value.String() == ""`?